yunwoo/LV_1/두-정수-사이의-합.js

```diff
@@ -0,0 +1,10 @@
+function solution(a, b) {
+  const sortedArr = [a, b].sort((a, b) => a - b);
+  let result = 0;
+
+  for (let i = sortedArr[0]; i <= sortedArr[1]; i++) {
+    result += i;
+  }
+
+  return result;
+}
```

Comment:
```suggestion
function solution(a, b) {
  const sortedArr = [a, b].sort((a, b) => a - b);
  let result = 0;

  for (let i = sortedArr[0]; i <= sortedArr[1]; i++) {
    result += i;
  }

  return result;
}
```
저는 Math.min, Math.max를 활용했는데 배열의 인덱스를 활용하는 윤우님의 코드 풀이 잘 봤습니다!

Reply:
코멘트 감사합니다👍 희지님 풀이도 잘 봤습니다 🙆‍♂️